README.md

```diff
@@ -127,6 +127,14 @@ The `cos_agent` interface is already supported by this Polkadot operator charm s
 
 Find more details on how to deploy and use COS [here](https://charmhub.io/topics/canonical-observability-stack/tutorials/instrumenting-machine-charms).
 
+#### Using an external relaychain node
+
+A parachain node can use an external relaychain node instead of the internal one. It's useful for scaling where multiple parachain nodes can share a relaychain node. It's also useful to get faster in sync since the parachain node does not have to sync a relaychain node by itself. This can be set with the service argument `--relay-chain-rpc-urls` that takes one or more weboscket URLs to relaychain nodes to to use. Setting multiple URLs is for fallback where the parachain node will try accessing the URLs in a round-robin fashion. Instead of setting this manually, the interface `rpc-url` can be used:
+
+    juju relate polkadot-relay:rpc-url polkadot-para:relay-rpc-url
```

Comment:
We are running on Juju 2.x, maybe soon 3.x, but I'm also thinking "what are everyone else running"? Since we have many downloads of this charm by now, we should make sure to think of all end users and not just ourselves.

```suggestion
    juju relate polkadot-relay:rpc-url polkadot-para:relay-rpc-url     # Juju 2.x
    juju integrate polkadot-relay:rpc-url polkadot-para:relay-rpc-url  # Juju 3.x
```